metals/src/main/scala/scala/meta/metals/bsp/BloopBspConnector.scala

```diff
@@ -0,0 +1,32 @@
+package scala.meta.metals.bsp
+
+import scala.meta.lsp.LanguageClient
+import scala.meta.lsp.LanguageServer
+import scala.meta.jsonrpc.BaseProtocolMessage
+import scala.meta.jsonrpc.Services
+import monix.eval.Task
+import monix.execution.Scheduler
+import java.nio.file.Files
+import java.nio.file.Paths
+import com.typesafe.scalalogging.LazyLogging
+
+class BloopBspConnector(services: Services)(implicit scheduler: Scheduler)
+    extends BspConnector
+    with LazyLogging {
+  private val socketPath = "/var/tmp/bsp"
```

Comment:
Windows users might be unhappy :)

Reply:
Aren't they always? 😇 

Jokes aside, my priority is making this work on my computer (I'm not there yet :( ) then I think supporting Windows users should be doable by using `--pipename`.